reproducibility_project/templates/crc.nd.sh

```diff
@@ -0,0 +1,15 @@
+{% extends "base_script.sh" %}
+{% block header %}
+#!/bin/bash
+#$ -N {{ id }}
+#$ -pe smp {{ np_global }}
+#$ -r n
+#$ -q long
+#$ -m abe
+#$ -M rsmith56@nd.edu
+
+conda activate mosdef-study38
+
+{% block tasks %}
+{% endblock %}
```

Comment:
Probably not too important, but do you want to start the `block tasks` before you close the `block header`?

If not, i would move one of the `endblock` commands to line 12

Reply:
I don't know all about how that works.  I only changed a few things from what Ryan DeFever had in another workflow.

Reply:
It seems to work as-is, and I'd rather not mess with it if it isn't broken, since I'm not familiar with how it works.

Reply:
Fair enough!